docker/docker-compose.dev.yml

```
@@ -130,3 +130,4 @@ volumes:
   wordlist:
   scan_results:
   ollama_data:
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)